Composer/packages/server/src/services/storage.ts

```diff
@@ -70,6 +72,20 @@ class StorageService {
   // return connent for file
   // return children for dir
   public getBlob = async (storageId: string, filePath: string, user?: UserIdentity) => {
+    if (filePath === thisPC) {
+      const diskNames = ['C:/', 'D:/', 'E:/', 'F:/', 'G:/', 'H:/', 'I:/', 'J:/', 'K:/'].filter(n => fs.existsSync(n));
```

Comment:
Couldn't disk names be arbitrary?

Reply:
We should call some system api, if not, use some commands should also be fine.

Reply:
Googled about this problem. Actually we cannot have an arbitrary names but we can change the label of disks. The Path C:/.../... won't change to Label:/.../... if we give it a label. So do we really need to show something like C:(label)/..?
Or are you suggesting I should remove the hard code here? I added a cmd script in env.ts which will get all disks names.